### PR TITLE
Simplify Piped installation flow in the quickstart guide

### PR DESCRIPTION
**Pull request by nghialv**
**What this PR does / why we need it**:

**Which issue(s) this PR fixes**:

Fixes #

**Does this PR introduce a user-facing change?**:

```release-note
NONE
```

### CODE REVIEW

File: manifests/piped/templates/configmap.yaml [hunk 1]
```diff
@@ -6,6 +6,22 @@ metadata:
...
+      projectID: quickstart
+      pipedID: {{ .Values.quickstart.pipedId }}
+      pipedKeyFile: /etc/piped-secret/piped-key
+      apiAddress: pipecd:8080
+      syncInterval: 1m
+      repositories:
+        - repoId: examples
+          remote: {{ .Values.quickstart.gitRepoRemote }}
+          branch: master
+{{- else }}
```
Comment (khanhtc1202) [inline manifests/piped/templates/configmap.yaml:23]: i see, this is nice 👍

File: manifests/piped/templates/configmap.yaml [hunk 2]
```diff
@@ -6,6 +6,22 @@ metadata:
...
   {{ .Values.config.fileName }}: |-
 {{ tpl .Values.config.data . | indent 4 }}
 {{- end }}
+{{- end }}
```
Comment (knanao) [inline manifests/piped/templates/configmap.yaml:27]: Unnecessary?

**Comment by khanhtc1202** [comment]
I think we should also update the quickstart docs under `/docs` directory since we should fix this quickstart docs not working issue before the next release.

**Comment by nghialv** [comment]
> I think we should also update the quickstart docs under /docs directory since we should fix this quickstart docs not working issue before the next release.

I don't think so since this new workaround will not work until the new release.
I mean the new helm chart is only delivered once a new version is released.

**Comment by khanhtc1202** [comment]
> > I think we should also update the quickstart docs under /docs directory since we should fix this quickstart docs not working issue before the next release.
> 
> I don't think so since this new workaround will not work until the new release. I mean the new helm chart is only delivered once a new version is released.

I see, so how about releasing next version as v0.23.1 ?

**Comment by khanhtc1202** [comment]
Nice work
/approve

**Pull request merged**